### ISSUE TITLE
Specify specific exceptions rather than generically catch them

### ISSUE THREAD

**Issue by rwoodpecker** [rwoodpecker/KoboBackup#23]
Per https://stackoverflow.com/questions/730764/how-to-properly-ignore-exceptions/736029#736029

**Comment by rwoodpecker** [comment]
Fixed and accidentally pushed directly to the dev branch.

@whatageareyouin I had to import the exceptions from tarfile into the main script, otherwise there would be no way to catch that specific exception, unless the exception is caught in the utils.py file in the tar function, rather than the try block in the main script. Thoughts?
 
https://github.com/rwoodpecker/KoboBackup/blob/c720437f7d4079ba565ec27acd91e999e38cb7bb/kobo_backup.py#L11
https://github.com/rwoodpecker/KoboBackup/blob/c720437f7d4079ba565ec27acd91e999e38cb7bb/kobo_backup.py#L92
